### PR TITLE
update deploy doc for 6.0 and gke

### PR DESCRIPTION
**Pull request by songbinliu**
Update documentation about deployment:

1. Use kubeturbo 6.0 image instead of 5.9;
2. Add documentation for GKE;

### CODE REVIEW

File: deploy/general-deploy/README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ In order to connect to your Turbonomic installation, a Kubeturbo configuration f
 Create a file called `config` in the `/etc/kubeturbo/` directory in the same node labeled in previous step, with the following contents:
 
 > The `<TURBONOMIC_SERVER_URL>` is typically `https://<TURBO_SERVER_IP>:443`
-> The `<TURBONOMIC_SERVER_VERSION>` is Turbonomic release version, e.g. `5.9.0` or `6.0.0`
+> The `<TURBONOMIC_SERVER_VERSION>` is Turbonomic release version, e.g. `5.9` or `6.0`
```
Comment (enlinxu) [inline deploy/general-deploy/README.md]: Isn't this the server version? Shouldn't that still be 6.0.0?

File: deploy/kubeturbo-for-gke/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+		"serverMeta": {
+            "version": "<TURBONOMIC_SERVER_VERSION>",
+		    "turboServer": "<TURBONOMIC_SERVER_URL>"
+		},
+		"restAPIConfig": {
+			"opsManagerUserName": "<TURBONOMIC_USERNAME>",
+			"opsManagerPassword": "<TURBONOMIC_PASSWORD>"
+		}
+	},
+	"targetConfig": {
```
Comment (enlinxu) [inline deploy/kubeturbo-for-gke/README.md]: I still think this part is a bit confusing. The username and password is confusing to the user.

File: deploy/kubeturbo-for-openshift/README.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ NAME           STATUS    AGE       LABELS
 
 A Kubeturbo config is required for Kubeturbo service to connect to Ops Manager server remotely. You need to specify correct **Turbonomic Server address**, **username** and **password**.
 > The `<SERVER_IP_ADDRESS>` is "**IP address of your ops manager**".
-> The `<TURBONOMIC_SERVER_VERSION>` is Turbonomic release version, e.g. `5.9.0` or `6.0.0`
+> The `<TURBONOMIC_SERVER_VERSION>` is Turbonomic release version, e.g. `59os` or `60os`
```
Comment (enlinxu) [inline deploy/kubeturbo-for-openshift/README.md]: Again, this is the config turbo version, not the image version.

File: deploy/general-deploy/README.md (hunk 1 above)
Comment (songbinliu) [reply deploy/general-deploy/README.md]: yea, you are right. I will change them back.

File: deploy/kubeturbo-for-gke/README.md (hunk 2 above)
Comment (songbinliu) [reply deploy/kubeturbo-for-gke/README.md]: Yea. it is. Should we add notes here?
Max is working on this [issue](https://github.com/turbonomic/kubeturbo/issues/101). We can update this documentation after the issue get fixed.

**Comment by songbinliu** [comment]
Sure. But I have to merge this [PR](https://github.com/turbonomic/kubeturbo/pull/124) first.
All the PRs are blocked by the go:tip check.

**Pull request merged**